Sources/Torrent Interface/Front End/Deluge.m

```diff
@@ -83,7 +83,7 @@ - (NSString *)getUserFriendlyAppendString
 
 - (NSString *)getURLAppendString
 {
-	return [[[FileHandler.sharedInstance webDataValueForKey:@"relative_path"] orSome:@""] stringByAppendingPathComponent:@"json"];
+	return [[[FileHandler.sharedInstance webDataValueForKey:@"relative_path"] orSome:@""] stringByAppendingPathComponent:@"/json"];
```

Comment:
That's weird, I thought `stringByAppendingPathComponent` would take care of there being a `/` there.

Reply:
Well, the left operand is `@""`. Then it moves upstream to `TorrentClient.m` and there's no path appender there...